Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -20,7 +20,49 @@ static public class DateTimeExtensions
     {
         if (nullableDateTime is DateTime dateTime)
             return DateTime.SpecifyKind(dateTime, DateTimeKind.Utc);
+        return null;
+    }
+
+    /// <summary>
+    /// Converts UTC DateTime to Eastern Standard Time 
+    /// List of time zones specified by TimeZoneInfo https://i.stack.imgur.com/zHzGt.png
+    /// </summary>
+    /// <param name="dateTime">DateTime with kind UTC</param>
+    /// <returns>Converted DateTime with kind Unspecified</returns>
+    static public DateTime ConvertUtcToEst(this DateTime dateTime) => TimeZoneInfo.ConvertTimeFromUtc(dateTime, TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time"));
```

Comment:
1. Can we extract either the string for EST or, preferrably, the entire EST timezone to a constant that can be re-used in the extensions?
2. Does this handle daylight-savings time?

Reply:
1. Yes, I can give it a string parameter then add `"Eastern Standard Time"` to Names.cs
2. Apparently yes, (we had this discussion offline) and I just tested it with a schedule of a series that passes through midnight 2023, March 12th

Reply:
> 2. Apparently yes, (we had this discussion offline) and I just tested it with a schedule of a series that passes through midnight 2023, March 12th

Just to confirm @amos-cha, you tested going past 2AM on March 12, 2023 right? In North America, the rollover happens at 2AM (meaning we skip straight from `2023-03-12 01:59` to `2023-03-12 03:00`).

Reply:
Edit: nope it breaks. 
Error: `The supplied DateTime represents an invalid time.  For example, when the clock is adjusted forward, any time in the period that is skipped is invalid.`

This is unfortunate... I'll have to look back into it

Reply:
> Edit: nope it breaks. Error: `The supplied DateTime represents an invalid time. For example, when the clock is adjusted forward, any time in the period that is skipped is invalid.`
> 
> This is unfortunate... I'll have to look back into it

Ah ok, so it does kinda handle Daylight Savings Time, but somewhat naively in that it doesn't automatically skip the "lost" hour. 

What is the Exception that this throws? You could catch exactly that exception and just handle the time change yourself.